app/views/devise/mailer/email_changed.html.erb

```diff
@@ -0,0 +1,7 @@
+<p>Hello <%= @email %>!</p>
```

Comment:
Optional: If you did not modify these files, you could remove the ones that are not customized and devise will fall back to the built-in views.

Reply:
Will icebox for sure!